mandatory/2-function-creation.js

```diff
@@ -15,26 +18,51 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    totalPrice -= totalPrice * 0.1;
```

Comment:
This is correct - can you think of another way of reducing the given amount by the percentage in a single calculation?